src/windowmanagement/Window/Window.tsx

```diff
@@ -0,0 +1,34 @@
+import { PropsWithChildren } from 'react';
+import { ErrorBoundary } from 'react-error-boundary';
+import { ScrollArea } from '@mantine/core';
+import { useElementSize } from '@mantine/hooks';
+
+import { fallbackRender } from '@/components/ErrorFallback/fallbackRenderer';
```

Comment:
```suggestion
import { fallbackRenderer } from '@/components/ErrorFallback/fallbackRenderer';
```
Rename this to match the file name?